### PR TITLE
Checking accessibility (pair-programming)

### PR DESCRIPTION
**Pull request by fed1k**
Checking accessibility :
   * Titles
   * alt texts for images
   * color contrast ratio

### CODE REVIEW

**Review by t-yanick — COMMENTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments below and  under the review._

- [ ] As it is a requirement for this project, please in your PR summary, you need to  give a report about the following checks;
- Page titles,
- Image text alternatives,
- Text headings,
- Color contrast,
- Resize,
- Interaction,
- Moving content,
- Multimedia,
- The basic structure of the page, against the items listed in this [article. ](https://www.w3.org/WAI/test-evaluate/preliminary/). 
- [ ] Each point in the list above needs to include an evaluation;
- If everything was ok, make a statement about that (e.g. "Page titles - no issues found").
- If you needed to change something, link to the corresponding line of code in your pull request changes.
In your PR summary, you have just listed the first three and have not given any report about them. Please kindly give a report about each (e.g. your page title is not descriptive, so you need to give report about that and then  link to the line of code where you have updated the page title). 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.css [hunk 1]
```diff
@@ -236,10 +236,12 @@ p {
 
 .skill2 {
   background: #e2505c;
+  background-color: hsl(354, 54%, 53%);
```
Comment (fed1k) [inline app.css:239]: * Color contrast made

File: app.css [hunk 2]
```diff
@@ -236,10 +236,12 @@ p {
...
 }
 
 .skill3 {
   background: #febf52;
+  background-color: hsl(37, 50%, 33%);
```
Comment (fed1k) [inline app.css:244]: * Color contrast made

File: app.css [hunk 3]
```diff
@@ -259,7 +261,7 @@ p {
   width: 180px;
   list-style: none;
   font-family: Poppins, serif;
-  color: #fff;
+  color: hsl(0, 0%, 41%);
```
Comment (fed1k) [inline app.css:265]: * Color contrast made

File: index.html [hunk 4]
```diff
@@ -14,7 +14,7 @@
         <!-- Navigation bar starts here -->
 
         <nav class="navbar">
-            <img id="menu-icon" src="images/ic-menu.svg" alt="">
+            <img id="menu-icon" src="images/ic-menu.svg" alt="Hamburger">
```
Comment (fed1k) [inline index.html:17]: * added text to `alt` attribute

File: index.html [hunk 5]
```diff
@@ -35,12 +35,12 @@ <h1>Firdavs Allamurotov</h1>
...
      <!-- Projects Section start here-->
-     <img id="pro-img" src="images/Group-117.png" alt="">
+     <img id="pro-img" src="images/Group-117.png" alt="Collection of projects">
      <section class="projects">
         <h2>Projects</h2>
-        <img id="img-dots" src="images/image-dots.png" alt="">
+        <img id="img-dots" src="images/image-dots.png" alt="part of design">
         <section class="card">
-            <img class="images1" src="images/Rectangle-21.png" alt="">
+            <img class="images1" src="images/Rectangle-21.png" alt="Laptop">
```
Comment (fed1k) [inline index.html:43]: * Added text to `alt` attribute

File: index.html [hunk 6]
```diff
@@ -51,7 +51,7 @@ <h2>Project name goes here</h2>
         </section>
 
         <section class="card">
-            <img class="images2" src="images/Rectangle-21-1.png" alt="">
+            <img class="images2" src="images/Rectangle-21-1.png" alt="Macbook">
```
Comment (fed1k) [inline index.html:54]: * Added text to `alt` attribute

File: index.html [hunk 7]
```diff
@@ -159,16 +159,16 @@ <h2>Get started</h2>
...
-            <a href="https://medium.com/" rel="noopener noreferrer"><i class="fab fa-medium-m"></i></a>
-            <a href="https://angel.co/" rel="noopener noreferrer"><i class="fab fa-angellist"></i></a>
+            <a title="stackoverflow-icon" href="https://stackoverflow.com/users/17635012/firdavs" rel="noopener noreferrer"><i class="fab fa-stack-overflow"></i></a>
+            <a title="github-icon" href="https://github.com/fed1k" rel="noopener noreferrer"><i class="fab fa-github"></i></a>
+            <a title="twitter-icon" href="https://twitter.com/?lang=ru" rel="noopener noreferrer"><i class="fab fa-twitter"></i></a>
+            <a title="linkedin-icon" href="https://www.linkedin.com/in/firdavs-allamurotov-12b60a226/" rel="noopener noreferrer"><i class="fab fa-linkedin"></i></a>
+            <a title="medium-icon" href="https://medium.com/" rel="noopener noreferrer"><i class="fab fa-medium-m"></i></a>
+            <a title="angelilist-icon" href="https://angel.co/" rel="noopener noreferrer"><i class="fab fa-angellist"></i></a>
```
Comment (fed1k) [inline index.html:171]: * Added text to `title` attribute

File: index.html [hunk 8]
```diff
@@ -73,7 +73,7 @@ <h2>Project name goes here</h2>
         </section>
     
         <section class="card">
-            <img class="images4" src="images/Rectangle-21-3.png" alt="">
+            <img class="images4" src="images/Rectangle-21-3.png" alt="Responsiveness pattern">
```
Comment (fed1k) [inline index.html:76]: * Added text to `alt` attribute

File: index.html [hunk 9]
```diff
@@ -84,7 +84,7 @@ <h2>Project name goes here</h2>
         </section>
     
         <section class="card">
-            <img class="images5" src="images/Rectangle-21-4.png" alt="">
+            <img class="images5" src="images/Rectangle-21-4.png" alt="Laptop">
```
Comment (fed1k) [inline index.html:87]: * Added text to `alt` attribute

File: index.html [hunk 10]
```diff
@@ -159,16 +159,16 @@ <h2>Get started</h2>
             <button class="form-button" type="submit">Start collaboration</button>
         </form>
         <div id="last-img-container">
-            <img id="last-img" src="images/image-geometry-7.png" alt="">
+            <img id="last-img" src="images/image-geometry-7.png" alt="geometry-shape">
```
Comment (fed1k) [inline index.html:162]: * Added text to `alt` attribute

File: index.html [hunk 11]
```diff
@@ -146,7 +146,7 @@ <h2>About</h2>
                     <li>Terminal</li>
                 </ul>
             </section>
-            <img id="dot" src="images/image-dots-with-teal.png" alt="">
+            <img id="dot" src="images/image-dots-with-teal.png" alt="dots-with-teals">
```
Comment (fed1k) [inline index.html:149]: * Added text to `alt` attribute

File: index.html [hunk 12]
```diff
@@ -95,7 +95,7 @@ <h2>Project name goes here</h2>
         </section>
     
         <section class="card">
-            <img class="images6" src="images/Rectangle-21-5.png" alt="">
+            <img class="images6" src="images/Rectangle-21-5.png" alt="Responsiveness pattern">
```
Comment (fed1k) [inline index.html:98]: * Added text to `alt` attribute

**Review by fed1k — COMMENTED** [review]
* Page titles - no issues found
* Text headings - no issues found
* Resize - no issues found
* Interaction - no issues found
* Multimedia - no issues found
* Moving content - no issues found
* Basic structure of the page - no issues found

File: index.html [hunk 13]
```diff
@@ -6,15 +6,15 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="app.css">
     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.0/css/all.css"/>
-    <title>Home page</title>
+    <title>Personal Portfolio</title>
```
Comment (moise10r) [inline index.html:9]: - [OPTIONAL] - Please re-write the title tag as the requirement say see the screenshot 👇

kkk

File: index.html [hunk 14]
```diff
@@ -6,15 +6,15 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (moise10r) [inline index.html:6]: - [ ] Kindly add the in your Pull Request :point_down:  just copy and past it there

pr

**Review by moise10r — APPROVED** [review]
<h1>Status: Approve  :champagne:  :+1: :confetti_ball: 

### **Hi @fed1k** , 

### **Congratulations! 🎉 Project Approved ✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 


Cheers and Happy coding!👏👏👏 💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**